docs/Cube.js-ClI/Using-the-Cube.js-CLI.md

```diff
@@ -8,49 +8,49 @@ menuOrder: 1
 The Cube.js command line interface (CLI) is used for various Cube.js workflows.
 It could help you in areas such as:
 
-* Creating a new Cube.js service;
-* Generating a schema based on your database tables;
-
-## Installation
-
-Cube.js CLI can be installed via NPM or Yarn.
-
-#### NPM
-```bash
-$ npm install -g cubejs-cli
-```
-
-#### Yarn
-```bash
-$ yarn global add cubejs-cli
-```
+- Creating a new Cube.js service;
+- Generating a schema based on your database tables;
 
 ## Quickstart
+
 Once installed, run the following command to create new Cube.js service
 
 ```bash
-$ cubejs create <project name> -d <database type>
+$ npx cubejs-cli create <project name> -d <database type>
 ```
 
-specifying the project name and your database using `-d` flag. Available options: 
+specifying the project name and your database using `-d` flag. Available
+options:
 
-* `postgres`
-* `mysql`
-* `mongobi`
-* `athena`
-* `redshift`
-* `bigquery`
-* `mssql`
-* `clickhouse`
-* `snowflake`
-* `presto`
+- `postgres`
+- `mysql`
+- `mongobi`
+- `athena`
+- `redshift`
+- `bigquery`
+- `mssql`
+- `clickhouse`
+- `snowflake`
+- `presto`
```

Comment:
@hassankhan Not something that should be addressed in this PR but worth mentioning. I wonder what's the order of items here (and in other parts of the docs). They look sorted by "subjective popularity" but I'm not sure. Maybe we can have them sorted alphabetically to make the list more accessible. Also, we can try to collect some stats either from our community or from the openly available data to know the popularity for sure.